catalyst/dl/callbacks/__init__.py

```diff
@@ -1,14 +1,8 @@
 # flake8: noqa
 
-from .checkpoint import CheckpointCallback, IterationCheckpointCallback
-from .criterion import (
-    CriterionAggregatorCallback, CriterionCallback,
-    CriterionOutputOnlyCallback
-)
+from catalyst.core.callback import *
+from catalyst.core.callbacks import *
```

Comment:
why are there two `catalyst.core.callback` and `catalyst.core.callbacks` (with `s` at the end)? what it for?